iam/api/tests.py

```diff
@@ -179,6 +179,96 @@ def _verify_cases(self, cases):
             self.assertEqual(token.userid, case['userid'])
             self.assertEqual(token.other_values, case['other_values'])
 
+class ApiTestHtmlEmail(TestCase):
+    def setUpTestData():
+        flush_db_load_fixture()
+
+    def setUp(self):
+        self.aeid_special = 1
+        u = User(username=test_data.admin['username'], email=test_data.admin['email'])
+        u.set_password(test_data.admin['password'])
+        u.save()
+        u.userdata.event = AuthEvent.objects.get(pk=1)
+        u.userdata.save()
+        self.user = u
+
+        self.admin_auth_data = dict(
+            email=test_data.admin['email'],
+            code="ERGERG")
+        c = Code(
+            user=self.user.userdata,
+            code=self.admin_auth_data['code'],
+            auth_event_id=self.aeid_special)
+        c.save()
+
+        u2 = User(username='noperm', email="noperm@sequentech.io")
+        u2.set_password("qwerty")
+        u2.save()
+        u2.userdata.save()
+
+        self.aeid_special = 1
+
+    def create_authevent(self, authevent):
+        c = JClient()
+        response = c.authenticate(self.aeid_special, self.admin_auth_data)
+        self.assertEqual(response.status_code, 200)
+        return c.post('/api/auth-event/', authevent)
+
+    def add_census(self, aeid):
+        c = JClient()
+        response = c.authenticate(self.aeid_special, self.admin_auth_data)
+        self.assertEqual(response.status_code, 200)
+
+        response = c.census(aeid, test_data.census_email_default)
+        self.assertEqual(response.status_code, 200)
+        response = c.get('/api/auth-event/%d/census/' % aeid, {})
+        self.assertEqual(response.status_code, 200)
+        r = parse_json_response(response)
+        self.assertEqual(len(r['object_list']), 4)
+
+    def test_create_authevent_html_email(self):
```

Comment:
Please add some more tests:
- checking that if allow_html_emails is disabled in django config, it doesn't work
- some other negative tests if you envision any other